dafni/data/outputs/causal_tests_results.json

```diff
@@ -42,9 +48,15 @@
       "outcome": "cum_vaccinated",
       "adjustment_set": [],
       "effect_measure": "coefficient",
-      "effect_estimate": 213111.93333333335,
-      "ci_low": 212755.15056812647,
-      "ci_high": 213468.71609854023
+      "effect_estimate": {
+        "max_doses": 213111.93333333335
+      },
+      "ci_low": [
+        212755.15056812647
+      ],
+      "ci_high": [
+        213468.71609854023
+      ]
```

Comment:
@jmafoster1 The estimated values are consistent and haven't changed, which is good, but for some reason, it's storing them as a dictionary now. Do you know why it would suddenly do this?

Reply:
Yes, Chris made this a thing ages ago in order to standardise the CTF outputs between numerical and categorical data. Categorical data will give you an effect estimate for each category of the variable. Doing it this way means we don't need to test datatypes everywhere.